lib/ExecutionControlEpic/TaggingFeature/Actions/FeatureLoad.js

```diff
@@ -0,0 +1,18 @@
+"use babel";
```

Comment:
I'm not sure this action is very useful, especially if it's only dispatched at startup, especially especially if it's only used for logging. If think it should be removed to reduce bloat.